src/System.Device.Gpio/System/Device/Gpio/PinVector32.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Gpio
+{
+    /// <summary>
+    /// 32-bit vector of pins and values.
+    /// </summary>
+    public struct PinVector32
```

Comment:
Let's get an issue filed with this proposal instead of having it on this PR. That way we can review it like we do for other proposals and once we have a design in mind we can submit a PR with the implementation

Reply:
I've created the issue #230 and moved the structs to \devices\Common pending resolution.